### PR TITLE
Cross post dialog

### PR DESCRIPTION
**Pull request by talhasch**

### CODE REVIEW

File: src/common/i18n/locales/en-US.json [hunk 1]
```diff
@@ -162,6 +162,7 @@
     "title": "Share"
   },
   "entry-menu": {
+    "cross-post": "Cross post in ...",
```
Comment (feruzm) [inline src/common/i18n/locales/en-US.json:165]: Just `Cross post`

File: src/common/i18n/locales/en-US.json [hunk 2]
```diff
@@ -175,6 +176,15 @@
     "promote": "Promote",
     "boost": "Boost"
   },
+  "cross-post": {
+    "title": "Cross Post",
+    "community-label": "Where to share this post:",
+    "community-placeholder": "Select a Community",
```
Comment (feruzm) [inline src/common/i18n/locales/en-US.json:182]: `Choose a community`

File: src/common/i18n/locales/en-US.json [hunk 3]
```diff
@@ -175,6 +176,15 @@
     "promote": "Promote",
     "boost": "Boost"
   },
+  "cross-post": {
+    "title": "Cross Post",
+    "community-label": "Where to share this post:",
```
Comment (feruzm) [inline src/common/i18n/locales/en-US.json:181]: Unnecessary, we don't need this line I think in UI.

File: src/common/i18n/locales/en-US.json [hunk 4]
```diff
@@ -175,6 +176,15 @@
...
+    "community-placeholder": "Select a Community",
+    "message-label": "Your message to the community:",
```
Comment (feruzm) [inline src/common/i18n/locales/en-US.json:183]: Unnecessary line, also can be removed from UI. But bottom of message input, please have informational text:

`Rewards are automatically declined for the cross post and discussions under cross post will be different and will follow different community guidelines.`

**Comment by talhasch** [comment]
@feruzm updated

File: src/common/i18n/locales/en-US.json [hunk 5]
```diff
@@ -175,6 +176,16 @@
...
     "boost": "Boost"
   },
+  "cross-post": {
+    "title": "Cross Post",
+    "community-label": "Where to share this post:",
+    "community-placeholder": "Choose a community",
+    "message-placeholder": "Start the conversation with a message",
+    "submit-label": "Confirm",
+    "published": "Post Published",
+    "no-subscription": "You don't have any community subscription to cross post.",
```
Comment (feruzm) [inline src/common/i18n/locales/en-US.json:186]: Let's add: Subscribe to some communities link `/communities`

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**